jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/milestones/CaseMilestoneListPresenter.java

```diff
@@ -29,13 +30,31 @@
 import org.uberfire.client.mvp.UberElement;
 
 import static org.jbpm.workbench.cm.client.resources.i18n.Constants.*;
+import static java.util.stream.Collectors.toList;
 
 @Dependent
 @WorkbenchScreen(identifier = CaseMilestoneListPresenter.SCREEN_ID)
 public class CaseMilestoneListPresenter extends AbstractCaseInstancePresenter<CaseMilestoneListPresenter.CaseMilestoneListView> {
 
+    public int currentPage = 0;
+    public static final int PAGE_SIZE = 10;
+
     public static final String SCREEN_ID = "Case Milestone List";
 
+    List<CaseMilestoneSummary> visibleCaseMilestones = new ArrayList<CaseMilestoneSummary>();
+
+    public int getPageSize() {
```

Comment:
`getPageSize` should probably be an abstract method at `AbstractPresenter` so each view would implement it accordantly.